pennylane_lightning/src/benchmarks/Bench_LinearAlgebra.cpp

```diff
@@ -42,12 +42,12 @@ static void create_random_cmplx_vector(benchmark::State &state) {
     }
 }
 BENCHMARK(create_random_cmplx_vector<float>)
-    ->RangeMultiplier(1l << 3)
```

Comment:
It is strange that `clang-tidy` didn't complain about this. Or don't we run `clang-tidy` for benchmark code?

Reply:
I think we never ran it in the original submissions. I would've thought we'd see the complaints earlier too.